apps/emqx/src/emqx_access_control.erl

```diff
@@ -224,8 +224,10 @@ check_authorization_cache(ClientInfo, Action, Topic) ->
             AuthzResult
     end.
 
-do_authorize(ClientInfo, Action, Topic) ->
+do_authorize(ClientInfo, Action, Topic0) ->
+    MountPrefix = emqx:get_config([mqtt, mount_prefix_for_authz], false),
```

Comment:
Q: Perhaps `emqx:get_config([authorization, include_mountpoint], false)`? "Mount prefix" sounds slightly odd to me because there's no option named "prefix" in schema (only in documentation).

Reply:
:checkered_flag: